engine/src/main/java/io/camunda/zeebe/engine/processing/processinstance/CreateProcessInstanceProcessor.java

```diff
@@ -55,6 +59,14 @@
 
   private static final Either<Rejection, Object> VALID = Either.right(null);
 
+  private static final Set<BpmnElementType> UNSUPPORTED_ELEMENT_TYPES =
+      Set.of(
+          BpmnElementType.START_EVENT,
+          BpmnElementType.SEQUENCE_FLOW,
+          BpmnElementType.BOUNDARY_EVENT,
+          BpmnElementType.UNSPECIFIED,
+          BpmnElementType.MULTI_INSTANCE_BODY);
```

Comment:
:x: We should allow the process instance creation at multi-instance elements (but no elements inside a multi-instance subprocess).  

Currently, we don't have a test case to verify that we can create on a multi-instance element. But I don't see why we should reject the creation on such an element.